eng/Versions.props

```diff
@@ -61,6 +61,7 @@
     <SystemIOCompressionTestDataVersion>5.0.0-beta.19608.5</SystemIOCompressionTestDataVersion>
     <SystemIOPackagingTestDataVersion>5.0.0-beta.19608.5</SystemIOPackagingTestDataVersion>
     <SystemNetTestDataVersion>5.0.0-beta.19608.5</SystemNetTestDataVersion>
+    <SystemPrivateRuntimeUnicodeDataVersion>5.0.0-beta.19610.1</SystemPrivateRuntimeUnicodeDataVersion>
```

Comment:
you need to add a dependency to eng/Version.Details.xml so that the version gets auto-updated.

Reply:
https://github.com/dotnet/runtime/pull/1731 is out. Thanks for catching this! :)